cachito/__init__.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import os
+from flask import Flask
+
+app = Flask(__name__)
+app.config.from_object('cachito.default_settings')
+app.config.from_envvar('CACHITO_SETTINGS')
+
+if not app.debug:
+    import logging
+    from logging.handlers import TimedRotatingFileHandler
+    # https://docs.python.org/3.6/library/logging.handlers.html#timedrotatingfilehandler
+    file_handler = TimedRotatingFileHandler(os.path.join(app.config['LOG_DIR'], 'cachito.log'),
```

Comment:
I don't think we want file logs. I think we want to stream them to stdout for OpenShift captures them.

Reply:
Removed it. Part of the cookiecutter stuff. Dropped it for now.